src/nvim/eval.c

```diff
@@ -10929,6 +10929,89 @@ static void f_getwininfo(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   }
 }
 
+// Dummy timer callback. Used by f_wait().
+static void dummy_timer_due_cb(TimeWatcher *tw, void *data)
+{
+  if (!uv_timer_get_repeat(&tw->uv)) {
```

Comment:
not needed, this was only for `interval=0` in a timer, and we require `interval>0` when present.

Reply:
I supposed that `interval` can be zero (i.e.: evaluate the condition as soon as possible). There are test cases with that. Is it bad performance-wise?

Reply:
It does not mean "as soon as possible" (we do that unconditionally already), but "constantly spin the CPU". Maybe it should be allowed, but certainly not recommended.